features/step_definitions/opportunities_steps.rb

```diff
@@ -63,8 +63,8 @@
     '//*[@class="search-result"]//*[@class="search-result-metadata"][2]//*[@class="search-result-metadata-item"][1]'
   )
   published_or_closed.each do |x|
-    if ['Closed', 'Unsuccessful', 'Cancelled'].include? status
-      x.text.should == status
+    if %w[Closed Unsuccessful Cancelled].include? status
```

Comment:
This array syntax might be a little confusing for people not used to Ruby.

Reply:
Ooo that looks weird 😹

Reply:
Do we need `Awarded` as well?

Reply:
Literal arrays are used by the Ruby community generally instead of the more traditional method. Will swap it back and also add the awarded.